test/RegistryTests.js

```diff
@@ -682,8 +683,32 @@ const runDynamicTestSuite = (suite_numOrgs, suite_servicesPerOrg, suite_reposPer
     });
 };
 
-// const testVector = [1,3,0,0];
-//
-// runDynamicTestSuite.apply(null, testVector);
+// ERC165 tests
+contract(`Registry ERC-165 test`, async (accounts) => {
+
+    it(`Validates the IRegistry ERC165 identifier`, async () => {
+        const identifier = generateInterfaceId(generateSelectorArray(IRegistry));
+
+        assert.equal(STRINGS.REGISTRY_ERC165_ID, identifier, "incorrect IRegistry ERC165 identifier");
+    });
+
+    // Registry will always have supportsInterface which will change the ID
+    it(`Validates that Registry and IRegistry have different identifiers`, async () => {
```

Comment:
NIT: this test seems superfluous

Reply:
this seems like a useful sanity check to me, wontfix